osu.Game/Rulesets/Judgements/DrawableJudgement.cs

```diff
@@ -65,13 +66,24 @@ private void load(OsuColour colours)
                 Anchor = Anchor.Centre,
                 Origin = Anchor.Centre,
                 RelativeSizeAxes = Axes.Both,
-                Child = new SkinnableDrawable(new GameplaySkinComponent<HitResult>(Result.Type), _ => JudgementText = new OsuSpriteText
+                Children =  new Drawable []
                 {
-                    Text = Result.Type.GetDescription().ToUpperInvariant(),
-                    Font = OsuFont.Numeric.With(size: 20),
-                    Colour = judgementColour(Result.Type),
-                    Scale = new Vector2(0.85f, 1),
-                }, confineMode: ConfineMode.NoScaling)
+                    new SkinnableDrawable(new GameplaySkinComponent<HitResult>(Result.Type), _ => JudgementText = new OsuSpriteText
+                    {
+                        Text = Result.Type.GetDescription().ToUpperInvariant(),
+                        Font = OsuFont.Numeric.With(size: 20),
+                        Colour = judgementColour(Result.Type),
+                        Scale = new Vector2(0.85f, 1),
+                    }, confineMode: ConfineMode.NoScaling),
+                    new SkinnableDrawable(new GameplaySkinComponent<HitDetail>(Result.Detail), _ => JudgementDetailText = new OsuSpriteText
+                    {
+                        Text = detailText(Result),
+                        Font = OsuFont.Numeric.With(size: 15),
+                        Y = -20,
```

Comment:
This -20 is kinda arbitrary, I may be missing a way to display this component above the other one nicely

Reply:
you're probably looking for a `FillFlowContainer`, check its usages for understanding on how to use it.

Reply:
FlowContainer with Direction = FillDirection.Vertical